uptime-kuma.json

```diff
@@ -597,134 +571,1545 @@
-              "viz": false
-            },
-            "insertNulls": false,
-            "lineInterpolation": "smooth",
-            "lineStyle": {
-              "fill": "solid"
-            },
-            "lineWidth": 1,
-            "pointSize": 5,
-            "scaleDistribution": {
-              "type": "linear"
-            },
-            "showPoints": "auto",
-            "spanNulls": 300000,
-            "stacking": {
-              "group": "A",
-              "mode": "none"
-            },
-            "thresholdsStyle": {
-              "mode": "dashed+area"
-            }
+            "mode": "thresholds"
           },
           "decimals": 0,
           "mappings": [],
           "min": 0,
+          "noValue": "0",
           "thresholds": {
             "mode": "absolute",
             "steps": [
               {
                 "color": "green",
                 "value": null
               },
-              {
-                "color": "yellow",
-                "value": 400
-              },
-              {
-                "color": "dark-orange",
-                "value": 600
-              },
               {
                 "color": "red",
-                "value": 1000
-              },
-              {
-                "color": "dark-red",
-                "value": 1500
+                "value": 80
               }
             ]
           },
-          "unit": "ms"
+          "unit": "short"
         },
         "overrides": []
       },
       "gridPos": {
-        "h": 14,
-        "w": 10,
-        "x": 14,
-        "y": 14
+        "h": 24,
+        "w": 3,
+        "x": 9,
+        "y": 5
       },
-      "id": 7,
+      "id": 28,
       "options": {
-        "legend": {
+        "colorMode": "none",
+        "graphMode": "none",
+        "justifyMode": "center",
+        "orientation": "auto",
+        "reduceOptions": {
           "calcs": [
-            "mean",
-            "min"
+            "last"
           ],
-          "displayMode": "table",
-          "placement": "right",
-          "showLegend": true,
-          "sortBy": "Mean",
-          "sortDesc": true
-        },
-        "tooltip": {
-          "mode": "single",
-          "sort": "none"
-        }
+          "fields": "",
+          "values": false
+        },
+        "showPercentChange": false,
+        "textMode": "value_and_name",
+        "wideLayout": true
       },
+      "pluginVersion": "10.4.1",
       "targets": [
         {
           "datasource": {
             "type": "prometheus",
-            "uid": "SUpEVGqnk"
+            "uid": "${datasource}"
           },
           "disableTextWrap": false,
           "editorMode": "builder",
-          "expr": "uptime_response_time{monitor_type=\"ping\", instance=\"$Instance\", job=\"$job\"}",
+          "exemplar": false,
+          "expr": "sort_desc(count by(monitor_type) (monitor_status{instance=\"$Instance\", job=\"$job\", monitor_type!=\"group\"}))",
```

Comment:
unclear if `monitor_type!=\"group\"` is correct here

Reply:
Yes. Ive changed the whole panel and now it shows the Services under Maintenance rather then the response-time for the ping-based services. Details of the different service types are now shown as separate Rows selectable via variable at the top.